README.md

```diff
@@ -41,3 +41,25 @@ Create a kubernetes manifest for a pod which will containa ToDo app container:
 1. `README.md` file should contain instructions on how to test ToDo application using the service `port-forward` command
 1. `README.md` should contain instruction on how to access an app using a NodePort Service
 1. Create PR with your changes and attach it for validation on a platform.
+
+### Run the following command to apply all manifests:
+```
+kubectl apply -f ./.infrastructure/namespace.yml
+kubectl apply -f ./.infrastructure/clusterip.yml
+kubectl apply -f ./.infrastructure/nodeport.yml
+kubectl apply -f ./.infrastructure/busybox.yml
+kubectl apply -f ./.infrastructure/todoapp-pod.yml
+```
+### Test an app by calling a ClusterIP service:
+Go to busybox shell:
+```
+kubectl -n todoapp exec -it busybox -- sh
+```
```

Comment:
Add URL where you will use ClusterIP service DNS
`curl http://todoapp-clusterip.todoapp.svc.cluster.local`

Reply:
added